libafl_intelpt/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "libafl_intelpt"
+version.workspace = true
+authors = ["Marco Cavenati <cavenatimarco@gmail.com>"]
+description = "Intel Processor Trace wrapper for libafl"
+repository = "https://github.com/AFLplusplus/LibAFL/"
+edition = "2021"
+license.workspace = true
+readme = "./README.md"
+keywords = ["fuzzing", "testing", "security", "intelpt"]
+categories = ["development-tools::testing", "no-std"]
+
+[features]
+default = ["std", "libipt"]
+std = ["libafl_bolts/std"]
+
+libipt = ["std", "dep:libipt"]
+
+[dev-dependencies]
+static_assertions = { workspace = true }
+
+[target.'cfg(target_os = "linux" )'.dev-dependencies]
+nix = { workspace = true }
+proc-maps = "0.4.0"
+
+[dependencies]
+#arbitrary-int = { version = "1.2.7" }
+#bitbybit = { version = "1.3.2" }
```

Comment:
Still survived here somehow? :) 
We can remove it later, or bring it back.
Again: you're very welcome to use bitbybit instead of the other bitfield thingy! Just please change the current occurances (I think it's only one or two) with the new crate then, so we don't have redundant crates

Reply:
I might be wrong but, maybe you missed my previous comment? :)

> I see, for the moment I changed the code using it and commented out the dependency.
> It was used to just build a constant so it's not a big deal. I will introduce it in the future with a new PR changing also the c2rust-bitfields.
> (In the future the PtConfig bitfield struct will potentially support more options)